kolibri/core/assets/src/api_resource.js

```diff
@@ -0,0 +1,68 @@
+const logging = require('loglevel');
+const rest = require('rest');
+const mime = require('rest/interceptor/mime');
+
+const Kolibri = require('kolibri');
+
+const client = rest.wrap(mime);
+
+class Model extends Object {
+  constructor(data, { resourceName, idKey, params } = { idKey: 'id', params: {} }) {
+    super(data);
+    this._resourceName = resourceName;
+    if (typeof this._resourceName !== 'string') {
+      throw new TypeError('resourceName must be defined');
+    }
+    this._idKey = idKey;
+    this._params = params;
+  }
+  fetch() {
+    return new Promise((resolve, reject) => {
+      client({ path: this._url(), params: this._params }).then((response) => {
+        Object.assign(this, response.entity);
+        resolve(this);
+      }, (response) => {
+        logging.error('An error occurred', response);
+        reject(response);
+      });
+    });
+  }
+  _url() {
+    return Kolibri.urls[`${this._resourceName}_detail`](this[this._idKey]);
```

Comment:
The template tag at the line above does it.

The settings that we have set for Django JS Reverse (https://github.com/ierror/django-js-reverse#options) uses the kolibriGlobal object to set the urls property on, rather than setting it directly on `window`.